pkg/provision/ec2.go

```diff
@@ -229,12 +242,13 @@ func (p *EC2Provisioner) lookupID(request HostDeleteRequest) (string, error) {
 	return "", fmt.Errorf("no host with ip: %s", request.IP)
 }
 
-// creteEC2SecurityGroup creates a security group for the exit-node
-func (p *EC2Provisioner) creteEC2SecurityGroup(controlPort int, pro string) (*string, *string, error) {
```

Comment:
Minor refactor included here to fix misspelling and introduce the new `vpcID` parameter.

Reply:
again, preserving the default of not needing to set IDs would be prefereable